mobile/library/objective-c/EnvoyConfiguration.h

```diff
@@ -43,6 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong) NSArray<NSString *> *virtualClusters;
 @property (nonatomic, strong) NSString *directResponseMatchers;
 @property (nonatomic, strong) NSString *directResponses;
+@property (nonatomic, strong) NSDictionary<NSString *, NSString *> *runtimeGuards;
```

Comment:
Objective-C is a bit limited here, but I think we can do better than `NSString` values by using `NSNumber` instead:

```suggestion
@property (nonatomic, strong) NSDictionary<NSString *, NSNumber *> *runtimeGuards;
```